### PR TITLE
Editorial batch

### PR DESCRIPTION
**Pull request by ptomato**
A few more minor things, and a fix for correctly uploading to codecov.io so that viewing source works in the codecov reports.

Closes: #2951 (cc @anba)

### CODE REVIEW

**Comment by ptomato** [comment]
Added a larger batch of @anba's requests onto here, since review hasn't started yet.

File: spec/plaintime.html [hunk 1]
```diff
@@ -516,9 +516,7 @@ <h1>
         1. Let _milliseconds_ be _ms2_ - _ms1_.
         1. Let _microseconds_ be _mus2_ - _mus1_.
         1. Let _nanoseconds_ be _ns2_ - _ns1_.
-        1. Let _norm_ be NormalizeTimeDuration(_hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_).
-        1. Assert: NormalizedTimeDurationAbs(_norm_).[[TotalNanoseconds]] &lt; nsPerDay.
```
Comment (Ms2ger) [inline spec/plaintime.html:520]: I'm not sure this is correct; `NormalizeTimeDuration` checks `abs(nanoseconds) ≤ maxTimeDuration`, and this is a check against `nsPerDay`, which is much smaller.

**Review by Ms2ger — APPROVED** [review]
Thanks!

File: spec/plaintime.html (hunk 1 above)
Comment (ptomato) [reply spec/plaintime.html:520]: Oops, you are totally right about this

**Pull request merged**